msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -482,4 +482,5 @@ items:
      :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/trigger-output.png" alt-text="Trigger Output"border="true":::
 
 - content: |
-    You've completed the tutorial to build notification bot!    
+   You've completed the tutorial to build a notification bot. You can extend your notification bot to add the command response and workflow features.
+   
```

Comment:
Can we say something like "You can follow the README file or go to https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/command-bot-in-teams to further extend your app?"